Documentation/ceph-csi-snapshot.md

```diff
@@ -99,3 +101,82 @@ kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/pvc-restore.yaml
 kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/snapshot.yaml
 kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/snapshotclass.yaml
 ```
+
+## CephFS Snapshots
+
+**NOTE: At present, there is a limit of 400 snapshots per cephFS filesystem. Also PVC cannot be deleted if it's having snapshots.
```

Comment:
How about moving this to a "limitations" section later in the doc and put it as bullet points? These don't seem like blocking issues that are important at the beginning of the section.
```suggestion
**NOTE: There is a limit of 400 snapshots per CephFS filesystem. Also the PVC cannot be deleted if it has snapshots.
```